src/clang_parser.cpp

```diff
@@ -159,10 +159,25 @@ void ClangParser::parse(ast::Program *program, StructMap &structs)
 
   struct utsname utsname;
   uname(&utsname);
-  std::string kernel_modules_dir = std::string("/lib/modules/") + utsname.release;
+  const char* env_kernel_modules_dir = ::getenv("BPFTRACE_KERNEL_SOURCE");
+  std::string kernel_modules_dir = env_kernel_modules_dir?
+					std::string(env_kernel_modules_dir):
+					std::string("/lib/modules/") +
+					utsname.release;
+
+  if (!is_dir(kernel_modules_dir)){
+	std::cerr << "WARNING: ("
+		  << kernel_modules_dir << ") is not a valid dir" << std::endl;
+	std::cerr << "Use environment variable BPFTRACE_KERNEL_SOURCE to setup"
+		     " a valid directory for kernel sources." << std::endl;
```

Comment:
s/kernel source/kernel headers/

Reply:
You are right, for consistency's sake BPFTRACE_KERNEL_HEADERS is the correct name.